iomgr/lib/dbengine/parser/AntlrHelpers.h

```diff
@@ -14,6 +14,24 @@ constexpr std::size_t kInvalidNodeType = std::numeric_limits<std::size_t>::max()
 
 namespace helpers {
 
+/**
+ * Convert double quote to single quote from string.
```

Comment:
Comment must be thought better. Term "double quote" means symbol `"`, not two `'`.
I suggest:
`Converts inplace two single quote into one single quote, as per SQL string literal syntax.`